### PR TITLE
PLANET-7301: Move Carousel Header block into master theme

### PR DESCRIPTION
**Pull request by GP-Dan-Tovbein**
Ref: [PLANET-7301](https://jira.greenpeace.org/browse/PLANET-7301)

[Demo page](https://www-dev.greenpeace.org/test-neptune/move-carousel-header-block-into-master-theme/)

# Description

Move the CarouselHeader block to master-theme

Related to [planet4-plugin-gutenberg-blocks#1171](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/1171)

### CODE REVIEW

**Comment by planet-4** [comment]
### Test instance is ready :rocket:

:new_moon: [neptune](https://www-dev.greenpeace.org/test-neptune) | [admin](https://www-dev.greenpeace.org/test-neptune/wp-admin/) | [blocks report](https://www-dev.greenpeace.org/test-neptune/wp-admin/?page=plugin_blocks_report) | [CircleCI](https://app.circleci.com/pipelines/github/greenpeace/planet4-test-neptune) | [composer-local.json](https://github.com/greenpeace/planet4-test-neptune/blob/main/composer-local.json)

:watch: 2023.12.12 09:38:48

File: src/Loader.php [hunk 1]
```diff
@@ -157,11 +157,12 @@ public static function add_blocks(): void
         new MasterBlocks();//NOSONAR
         new Blocks\GuestBook();//NOSONAR
 
-        // Load extra code necessary for our beta blocks.
         if (!BetaBlocks::is_active()) {
             return;
         }
 
+        new Blocks\CarouselHeader();//NOSONAR
```
Comment (GP-Dan-Tovbein) [inline src/Loader.php:164]: https://greenpeace-gpi.slack.com/archives/C0160MX64AG/p1702402413013349

File: assets/src/scss/blocks/CarouselHeader/CarouselHeaderStyle.scss [hunk 2]
```diff
@@ -0,0 +1,696 @@
...
+      @include x-large-and-up {
+        width: #{480 + ($text-slide-offset * 2)};
+      }
+
+      overflow: hidden;
+    }
+
+    .caption-overlay {
+      display: none;
+      background: url("../../images/carousel-blurred-overlay.png");
```
Comment (lithrel) [inline assets/src/scss/blocks/CarouselHeader/CarouselHeaderStyle.scss:365]: you're missing this image

**Review by lithrel — APPROVED** [review]
:+1:

**Pull request merged**